discojs/src/models/gpt/layers.spec.ts

```diff
@@ -0,0 +1,308 @@
+import * as tf from '@tensorflow/tfjs';
+import { expect } from 'chai';
+import { GELU, LMEmbedding, Range, MLP, MLPConfig, CausalSelfAttention, CausalSelfAttentionConfig } from './layers.js';
+
+describe('GPT Layers', function () {
+  // GELU Layer tests
+  describe('GELU Layer', function () {
+
+    afterEach(() => {
+      // Dispose of variables to avoid name collisions in subsequent tests.
+      tf.disposeVariables();
+    });
+
+    it('should compute GELU activation correctly for known inputs', async function () {
+      const geluLayer = new GELU();
+
+      const input: tf.Tensor1D = tf.tensor1d([0, 1, -1, 2, -2]);
```

Comment:
`tf.tensor1d` already returns a `tf.Tensor1D`

```suggestion
      const input = tf.tensor1d([0, 1, -1, 2, -2]);
```